import_token.go

```diff
@@ -133,6 +134,52 @@ func encryptNonce(key, value, iv string) (string, string, error) {
 	return base64.StdEncoding.EncodeToString(cipherText), iv, nil
 }
 
+// EncryptNonceWithCBC encrypts the nonce using the user's key-material
+// with CBC encrypter, it requires padding, here pkcs7 padding is used.
+// This is needed for Hyper Protect Crypto Services, since it supports only
+// CBC Encryption.
```

Comment:
So the padding part is a bit confusing here, as it says that it requires padding but says that pkcs7 padding is used.

Maybe change this to EncryptNonceWithCBCPAD and in the description say that padding will be added by the function and the user doesn't need to add any.

Reply:
CBC Encryption needs a padded value to encrypt, we use pkcs7 to do the padding, maybe that comment was too detailed to make the reader confused, will remove that part from the comment since the client should not care about it.